kingfisher_scrapy/extensions/files_store.py

```diff
@@ -45,10 +45,16 @@ def spider_opened(self, spider):
             path = os.path.join(self.relative_crawl_directory(spider), 'scrapyd-job.txt')
             self._write_file(path, spider._job)
 
-    def spider_closed(self, spider):
+    def spider_closed(self, spider, reason):
+        if reason not in ('finished', 'sample') or spider.pluck:
+            return
+
         path = os.path.join(self.directory, self.relative_crawl_directory(spider))
 
-        message = f'The data is available at: {path}'
+        if not os.path.exists(path):
+            message = 'SOMETHING FAILED AND NO DATA WAS DOWNLOADED. CHECK THE LOGS FOR DETAILS'
```

Comment:
We can remove "CHECK THE LOGS FOR DETAILS", since this message is printed in the log (at the end).

```suggestion
            message = 'Something went wrong. No data was downloaded.'
```